components/notebook-controller/Makefile

```diff
@@ -25,6 +28,9 @@ check-license:
 
 # Run tests
 test: generate fmt vet manifests check-license
+	mkdir -p ${ENVTEST_ASSETS_DIR}
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh
```

Comment:
Can you pin the script at a particular commit?
This way, we can be sure an update in this external repo doesn't break us.

Reply:
Thanks, I will do that. 👍